de.tu_bs.cs.isf.e4cf.compare.data_structures/src/de/tu_bs/cs/isf/e4cf/compare/data_structures/io/reader/Visitor.java

```diff
@@ -0,0 +1,989 @@
+package de.tu_bs.cs.isf.e4cf.compare.data_structures.io.reader;
+
+import de.tu_bs.cs.isf.e4cf.compare.data_structures.impl.NodeImpl;
+import de.tu_bs.cs.isf.e4cf.compare.data_structures.interfaces.Node;
+import javafx.scene.effect.Lighting;
+
+import com.github.javaparser.ast.visitor.*;
+import com.github.javaparser.ast.body.*;
+import com.github.javaparser.ast.expr.*;
+import com.github.javaparser.ast.stmt.*;
+import com.github.javaparser.ast.type.*;
+
+import java.util.Arrays;
+
+import com.github.javaparser.JavaParser;
+import com.github.javaparser.ast.*;
+
+/**
+ * Part of the custom visitor class. This class extends VoidVisitorAdapter
+ * (https://www.javadoc.io/doc/com.github.javaparser/javaparser-core/latest/com/github/javaparser/ast/visitor/VoidVisitorAdapter.html).
+ * 
+ * @author Serkan Acar
+ * @author Pascal Blum
+ * @author Paulo Haas
+ * @author Hassan Smaoui
+ *
+ */
+public class Visitor extends VoidVisitorAdapter<Node> {
+	/**

```

Comment:
Are those comments generated? I would expect comments such as, "This method process the compilation unit and is called ...